src/edge/importer.py

```diff
@@ -126,17 +129,16 @@ def build_fragment(self):
                     original_chunk_size -= chunk_size_limit
                 new_chunk_sizes.extend(divided_chunks)
         chunk_sizes = new_chunk_sizes
+        print('%d chunks' % (len(chunk_sizes),))
 
+        prev = None
+        flen = 0
```

Comment:
how about rename as fragement_len? just to be consistent with other variable names regarding len.